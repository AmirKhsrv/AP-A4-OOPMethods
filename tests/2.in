show_sorted_shelf 33 want_to_read HolidayHorror
best_reviewer
recommend_second_approach 34
recommend_first_approach 129
credit 90
show_sorted_shelf 61 currently_reading Fiction
credit 64
show_sorted_shelf 63 want_to_read Nonfiction
best_book
show_sorted_shelf 83 want_to_read Fantasy
show_author_info 127
best_book
credit 14
best_reviewer
