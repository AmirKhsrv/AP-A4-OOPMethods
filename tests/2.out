id: 268
Title: Angels Flight
Genre: Mystery
Author: Michael Connelly
***
id: 233
Title: Beauty's Release
Genre: Adult Fiction
Author: A.N. Roquelaure
***
id: 113
Title: Ghost in the Shell
Genre: Sequential Art
Author: Masamune Shirow
***
id: 128
Title: Grundish and Askew
Genre: Fiction
Author: Lance Carbuncle
***
id: 127
Title: Legend
Genre: Young Adult
Author: Marie Lu
***
id: 144
Title: Lies My Teacher Told Me: Everything Your American History Textbook Got Wrong
Genre: History
Author: James W. Loewen
***
id: 217
Title: Space Viking
Genre: Science Fiction
Author: H. Beam Piper
***
id: 61
Title: The Bronze Horseman
Genre: Historical
Author: Paullina Simons
***
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
0.005812
id: 281
Title: Some Are Sicker Than Others
Genre: Fiction
Author: Andrew Seaward
***
id: 223
Title: Demon Kissed
Genre: Young Adult
Author: H.M. Ward
***
id: 395
Title: Dirty Ross
Genre: Romance
Author: Sandrine Gasq-Dion
***
id: 64
Title: Flawless
Genre: Young Adult
Author: Lara Chapman
***
id: 179
Title: Ghostgirl
Genre: Fantasy
Author: Tonya Hurley
***
id: 360
Title: Steve Jobs
Genre: Biography
Author: Walter Isaacson
***
id: 306
Title: The Madness Underneath
Genre: Young Adult
Author: Maureen Johnson
***
id: 136
Title: Washington's Crossing
Genre: History
Author: David Hackett Fischer
***
0.004491
id: 356
Title: The Shock Doctrine: The Rise of Disaster Capitalism
Genre: Nonfiction
Author: Naomi Klein
***
id: 373
Title: A Shadow in Summer
Genre: Fantasy
Author: Daniel Abraham
***
id: 254
Title: Caps for Sale: A Tale of a Peddler Some Monkeys and Their Monkey Business
Genre: Childrens
Author: Esphyr Slobodkina
***
id: 128
Title: Grundish and Askew
Genre: Fiction
Author: Lance Carbuncle
***
id: 334
Title: Honey Queen
Genre: Young Adult
Author: Christina Mercer
***
id: 15
Title: Into the Wilderness
Genre: Historical
Author: Sara Donati
***
id: 338
Title: Jurassic Park
Genre: Science Fiction
Author: Michael Crichton
***
id: 151
Title: Mist of Midnight
Genre: Historical
Author: Sandra Byrd
***
id: 291
Title: Of Poseidon
Genre: Fantasy
Author: Anna Banks
***
id: 57
Title: Sleeping Stone
Genre: Romance
Author: Alexa Snow
***
id: 130
Title: Song to Wake to
Genre: Fantasy
Author: J.D. Field
***
id: 239
Title: Stories of Your Life and Others
Genre: Science Fiction
Author: Ted Chiang
***
id: 34
Title: The Maid of Fairbourne Hall
Genre: Historical
Author: Julie Klassen
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 226
Title: Asura: Tale Of The Vanquished
Genre: Fantasy
Author: Anand Neelakantan
***
id: 400
Title: Dragon Bound
Genre: Fantasy
Author: Thea Harrison
***
id: 115
Title: The High King
Genre: Fantasy
Author: Lloyd Alexander
***
id: 289
Title: Antigone
Genre: Plays
Author: Jean Anouilh
***
id: 385
Title: Conquest
Genre: Romance
Author: S.J. Frost
***
id: 59
Title: Impossible
Genre: Young Adult
Author: Nancy Werlin
***
id: 43
Title: Soulless
Genre: Romance
Author: T.M. Frazier
***
id: 383
Title: Yes Means Yes!: Visions of Female Sexual Power and A World Without Rape
Genre: Feminism
Author: Jaclyn Friedman
***
id: 127
Name: Tracie Peterson
Year of Birth: 1955
Place of Birth: Central African Republic
Member Since: February 17 2013
Genres: Historical, Religion Spirituality
Books:
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
0.012376
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
